packages/design-system-react/src/components/button-primary/ButtonPrimary.stories.tsx

```diff
@@ -0,0 +1,180 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { ButtonPrimarySize } from '.';
+import { IconName } from '..';
+import { ButtonPrimary } from './ButtonPrimary';
+import README from './README.mdx';
+
+const meta: Meta<typeof ButtonPrimary> = {
+  title: 'React Components/ButtonPrimary',
+  component: ButtonPrimary,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+      description:
+        'Required prop for the content to be rendered within the ButtonPrimary',
+    },
```

Comment:
Love that you add these descriptions to the controls. Just curious, do you know if there's a way we can reuse this description across types, readme, and stories?

Reply:
Great question! Not that I know of something to look into.